packages/connect/src/data/defaultFeeLevels.ts

```diff
@@ -113,7 +113,7 @@ const SOLANA_FEE_INFO: FeeInfoWithLevels = {
     blockTime: -1, // unknown
     defaultFees: [{ label: 'normal', feePerUnit: '5000', blocks: -1 }],
     minFee: 5000,
-    maxFee: 5000,
+    maxFee: 100000000, // arbitrary
```

Comment:
Is this a required parameter? if not I would drop it so it does not kick our ass in the future. What is the purpose of this field anyway?

Reply:
it is mandatory and since it's uncapped I plan to bring this up to the trezor guys in review